extensions/amp-analytics/0.1/events.js

```diff
@@ -380,6 +399,78 @@ export class CustomEventTracker extends EventTracker {
   }
 }
 
+class AmpStoryEventTracker extends EventTracker {
+  /**
+   * @param {!./analytics-root.AnalyticsRoot} root
+   */
+  constructor(root) {
+    super(root);
+
+    /** @private {?Observable<!Event>} */
+    this.sessionObservable_ = new Observable();
+
+    /** @private {?function(!Event)} */
+    this.boundOnSession_ = this.sessionObservable_.fire.bind(
+      this.sessionObservable_
+    );
+
+    Object.keys(StoryAnalyticsEvent).forEach(key => {
+      this.root
+        .getRoot()
+        .addEventListener(StoryAnalyticsEvent[key], this.boundOnSession_);
+    });
+  }
+
+  /** @override */
+  dispose() {
+    const root = this.root.getRoot();
+    Object.keys(StoryAnalyticsEvent).forEach(key => {
+      root.removeEventListener(StoryAnalyticsEvent[key], this.boundOnSession_);
+    });
+    this.boundOnSession_ = null;
+    this.sessionObservable_ = null;
+  }
+
+  /** @override */
+  add(context, eventType, config, listener) {
+    const storySpec = config['storySpec'] || {};
+    userAssert(!config['selector'], 'Selector not supported in AmpStorySpec');
+    const selector = ':root';
+    const selectionMethod = config['selectionMethod'] || null;
+    const targetReady = this.root.getElement(
```

Comment:
nit: Since the selector is always `:root` I don't think we need `selectionMethod`, and we can use the `this.root.getRootElement()` instead.

Reply:
Oh cool!